scripts/project.py

```diff
@@ -46,15 +48,14 @@ def execute(self):
         with tempfile.TemporaryDirectory() as d:
             tmpdir = pathlib.Path(d)
 
-            logging.info("Compiling project")
+            logging.warning("Compiling project")
```

Comment:
Why is this a warning? Looks like normal operation, doesn't it?

Reply:
Because with `info()`, it is not output with default verbosity. However, since the truffle compilation can easily take 5-10 seconds, I thought it might be interesting to provide a little information about the reason for the delay in the progress information.